xarray/core/dataarray.py

```diff
@@ -2113,6 +2113,15 @@ def rank(self, dim, pct=False, keep_attrs=False):
         ds = self._to_temp_dataset().rank(dim, pct=pct, keep_attrs=keep_attrs)
         return self._from_temp_dataset(ds)
 
+    def isin(self, test_elements):
```

Comment:
Great. Anyone recall the newest way to run across Variables in a Dataset (i.e. more standard than a loop)? 

I'm sure I saw it in a recent PR but can't find it, either in the code or the recent PRs

Otherwise loop is fine

Reply:
`apply_ufunc` can handle an xarray.Dataset as an argument.

Are you thinking of using a Dataset for test_elements as well?